src/js/core/checkTryCount.js

```diff
@@ -0,0 +1,16 @@
+import { $ } from '../util/dom.js';
+
+export const getTryCount = e => {
+  e.preventDefault();
+  const tryCountInput = $('#try-count-input').value;
+  if (!isTryCountPositiveNumber(tryCountInput)) {
+    return tryCountInput;
```

Comment:
음.. 이건 저라면 `tryCount` 라고 네이밍을 할 거 같아요. `getTryCountInput` 이라고 이름 안지은 것처럼용. Input 이 있으면 해당 변수는 InputElement 인가?라는 생각이 드네요. 



```suggestion
    return tryCount;
```


> 더욱 개인의 취향을 보태면 `countOfTry` ..!

Reply:
일단 tryCount로 수정했습니다!